src/FileCookieJar.php

```diff
@@ -23,14 +23,18 @@ final class FileCookieJar implements CookieJar
     /** @var Mutex */
     private $mutex;
 
-    public function __construct(string $storagePath, ?Mutex $mutex = null)
+    /** @var bool */
+    private $persistSessionCookies;
+
+    public function __construct(string $storagePath, ?Mutex $mutex = null, bool $persistSessionCookies = false)
```

Comment:
I'd rather add a method for this, boolean parameters except for setters are usually a bad idea.

Reply:
ok, changed